source/hp/refinement.cc

```diff
@@ -1059,6 +1044,38 @@ namespace hp
 
       return levels_changed;
     }
+
+
+
+    template <int dim, int spacedim>
+    unsigned int
+    copy_future_fe_indices(const DoFHandler<dim, spacedim> &dof_handler_source,
+                           DoFHandler<dim, spacedim> &dof_handler_destination)
+    {
+      Assert(dof_handler_source.get_triangulation().n_active_cells() ==
+               dof_handler_destination.get_triangulation().n_active_cells(),
+             ExcMessage(
+               "Both DoFHandler objects must work on the same Triangulation!"));
```

Comment:
```suggestion
      Assert(&dof_handler_source.get_triangulation() ==
               &dof_handler_destination.get_triangulation(),
             ExcMessage(
               "Both DoFHandler objects must work on the same Triangulation!"));
```

Reply:
Ah yes, comparing addresses is smarter :)